src/dataService/tagDataService.ts

```diff
@@ -56,7 +56,9 @@ export class TagDataService {
         this.db.raw(`TO_BASE64(tag) as id`),
         this.db.raw('GROUP_CONCAT(item_id) as savedItems'),
         this.db.raw('UNIX_TIMESTAMP(MIN(time_added)) as _createdAt'),
-        this.db.raw('UNIX_TIMESTAMP(MAX(time_updated)) as _updatedAt'),
+        this.db.raw(
+          'UNIX_TIMESTAMP(MAX(COALESCE(time_updated, time_added))) as _updatedAt'
```

Comment:
reading from PR description that we don't update the `time_updated` for item_tags table: wondering if we can just read `time_added`?

Reply:
I'm not 100% sure that it's never updated, but probably